features/utils-wp.feature

```diff
@@ -79,6 +86,18 @@ Feature: Utilities that depend on WordPress code
       """
     And save STDOUT as {DEFAULT_STDOUT}
 
+    When I run `wp --require=table_names.php get_table_names --all-tables --views-only`
+    Then STDOUT should contain:
+      """
+      posts_view
+      """
+
+    When I run `wp --require=table_names.php get_table_names --all-tables --base-tables-only`
+    Then STDOUT should not contain:
+      """
+      posts_view
```

Comment:
This should ensure the tables are actually being displayed.
If the output is a list, you can use a notation like follows:
```
   Then STDOUT should be a table containing rows:
      | name          |
      | xx_wp_posts   |
      | wp_xx_posts   |
      | wp_posts_xx   |
      | wp_categories |
```